schema.sql

```diff
@@ -306,4 +306,57 @@ VALUES
     ('ZM', 'Zambia'),
     ('ZW', 'Zimbabwe');
 
+INSERT INTO beta.plant
+    (plant_id, botanist_id, species_id, country_id)
+VALUES
+    (0, 1, 1, 24),
+    (1, 2, 2, 185),
+    (2, 1, 3, 128),
+    (3, 3, 4, 24),
+    (4, 1, 5, 53),
+    (5, 1, 6, 77),
+    (6, 3, 7, 32),
+    (8, 3, 8, 42),
+    (9, 2, 9, 65),
+    (10, 2, 10, 43),
+    (11, 2, 11, 185),
+    (12, 3, 12, 185),
+    (13, 3, 13, 65),
+    (14, 2, 14, 185),
+    (15, 2, 15, 177),
+    (16, 2, 16, 185),
+    (17, 1, 17, 78),
+    (18, 1, 18, 65),
+    (19, 2, 19, 23),
+    (20, 1, 20, 163),
+    (21, 3, 21, 24),
+    (22, 2, 8, 65),
+    (23, 3, 22, 65),
+    (24, 1, 23, 85),
+    (25, 2, 24, 165),
+    (26, 1, 25, 3),
+    (27, 1, 26, 182),
+    (28, 1, 27, 185),
+    (29, 2, 28, 85),
+    (30, 1, 29, 99),
+    (31, 2, 30, 36),
+    (32, 2, 31, 65),
+    (33, 1, 32, 77),
+    (34, 2, 33, 163),
+    (35, 1, 34, 35),
+    (36, 2, 35, 185),
+    (37, 3, 36, 77),
+    (38, 2, 37, 171),
+    (39, 1, 38, 85),
+    (40, 3, 39, 61),
+    (41, 1, 40, 77),
+    (42, 1, 41, 26),
+    (44, 2, 42, 112),
```

Comment:
missing plant_id 43 - is this expected?

Reply:
good spot! Yeah apparently its on loan to another museum (according to the endpoint)